semantic/src/site/elements/icon.overrides

```diff
@@ -1,3 +1,12 @@
 /*******************************
          Site Overrides
 *******************************/
+
+i.icon {
+  color: @darkGrey;
+}
+
+link.icon {
+  font-size: @fontXXsmall;
+  font-weight: @semiboldWeight;
+}
```

Comment:
(Similar to my comment below): You should be able to just over-ride variables to achieve this, no need to re-write class definitions.

Reply:
After talking in person. Done: 43e1624